modular_citadel/code/modules/reagents/chemistry/recipes/fermi.dm

```diff
@@ -591,21 +591,21 @@
 /datum/chemical_reaction/fermi/zeolites
 	name = "Zeolites"
 	id = /datum/reagent/fermi/zeolites
-	results = list(/datum/reagent/fermi/zeolites = 5) //We make a lot! - But it's now somewhat dangerous, and needs a bit of uranium to catalyze the reaction
+	results = list(/datum/reagent/fermi/zeolites = 5) //We make a lot! - But it's now somewhat dangerous, and needs a bit of gold to catalyze the reaction
 	required_reagents = list(/datum/reagent/medicine/potass_iodide = 1, /datum/reagent/aluminium = 1, /datum/reagent/silicon = 1, /datum/reagent/oxygen = 1)
-	required_catalysts = list(/datum/reagent/uranium = 5)
+	required_catalysts = list(/datum/reagent/gold = 5)
 	//FermiChem vars:
 	OptimalTempMin 	= 500
 	OptimalTempMax 	= 750
 	ExplodeTemp 	= 850
-	OptimalpHMin 	= 2.8
-	OptimalpHMax 	= 5 //2.2 ph levels of optimal ph zone - centered at 3.9 - ph of ingredients mixed at equal values is 9.55; ph of result is 8.
-	ReactpHLim 		= 4
+	OptimalpHMin 	= 4.8
+	OptimalpHMax 	= 7
+	ReactpHLim 		= 5
 	//CatalystFact 	= 0
 	CurveSharpT 	= 1.5
 	CurveSharppH 	= 3
-	ThermicConstant = 7
+	ThermicConstant = 5
 	HIonRelease 	= -0.15
 	RateUpLim 		= 4
-	PurityMin 		= 0.5 //Good luck.
+	PurityMin 		= 0.4
```

Comment:
This WILL cause bad things to happen, please adjust the formula of /datum/reagent/fermi/zeolites/on_mob_life() aswell if you are going to change purity minimum.

Reply:
ill revert this change then good eye!